### PR TITLE
tell people about your changes

### PR DESCRIPTION
**Pull request by dotcode**

### CODE REVIEW

**Comment by andrewmee** [comment]
(waits for you to forget to add this to the #frontend channel ;) )

**Comment by joseluisbolos** [comment]
I would argue against this in the way that it's currently written. We can set up notifications in Slack so they happen automatically. No one should have to copy and paste stuff!

**Comment by umaar** [comment]
As some of us may not catch up with the frontend slack channel in it's entirety all the time, relying on email notifications via Github notifications guarantees we're alerted of changes. We can even use smart email client filters to label the email in an interesting way when a merge occurs. Manual copy/paste is just another thing to remember and can be forgotten at times.

**Comment by dotcode** [comment]
@joseluisbolos quite right. I don't know what I was thinking. Can you set this up? Will rewrite this PR appropriately.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -25,3 +25,8 @@ Please create a new short-lived feature branch for your changes, then create a p
 Include a summary of your pull request when you open it. We prefer to use commit messages to explain _what_ a change will do, and pull request summaries to explain _why_ the change was made. [Read further guidelines on commit messages](practices/git.md) - many of these guidelines can also apply to pull request summaries!
 
 If you work at Springer Nature, please remember to keep discussion inside the issues and pull requests, avoiding Slack, hallway conversations etc. Remember that this repo is public, and the discussions we have can be of benefit to people apart from us.
+
+### Tell people about your changes
+
+We hope work committed here will be of interest to the wider development community. It's _definitely_ relevant to all Front-End developers at Springer Nature. If you work at Springer Nature, once you've had work merged, _message the #frontend Slack room_, providing a link to the pull request (e.g. [add ToV and plain english guidelines](https://github.com/springernature/frontend-playbook/pull/129)).
```
Comment (danyalaytekin) [inline CONTRIBUTING.md]: `Front-End` is inconsistent with `Front End` on line 1.

Comment (dotcode) [inline CONTRIBUTING.md]: Will change Front End to Front-End in subsequent commit.

**Comment by joseluisbolos** [comment]
@dotcode Done.

**Comment by dotcode** [comment]
@umaar yeah, the "manual" thing is a bit 1996 (bear with me, I'm old and tired).

Re: relying _solely_ on one method of communication (such as GitHub notifications) - I'm not a fan of that. Changes to the playbook affect all FEDs here, so it would be nice to broadcast them on  all channels. Some people don't pay attention to GitHub notifications (they're evil, obvs).

Let's go with what Jose suggested, and I'll see if I can rewrite this to make it worth opening in the first place :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -26,7 +26,8 @@ Include a summary of your pull request when you open it. We prefer to use commit
 
 If you work at Springer Nature, please remember to keep discussion inside the issues and pull requests, avoiding Slack, hallway conversations etc. Remember that this repo is public, and the discussions we have can be of benefit to people apart from us.
 
-### Tell people about your changes
+### Publicising your changes
 
-We hope work committed here will be of interest to the wider development community. It's _definitely_ relevant to all Front-End developers at Springer Nature. If you work at Springer Nature, once you've had work merged, _message the #frontend Slack room_, providing a link to the pull request (e.g. [add ToV and plain english guidelines](https://github.com/springernature/frontend-playbook/pull/129)).
+Changes to the playbook are relevant to all Front-End developers at Springer Nature, which is why once you've had work merged, the #frontend Slack room is notified, providing a link to the pull request.
```
Comment (hollsk) [inline CONTRIBUTING.md]: PASSIVE VOICE ALERT

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -28,6 +28,6 @@ If you work at Springer Nature, please remember to keep discussion inside the is
 
 ### Publicising your changes
 
-Changes to the playbook are relevant to all Front-End developers at Springer Nature, which is why once you've had work merged, the #frontend Slack room is notified, providing a link to the pull request.
+The #frontend Slack room is notified when work is merged, because changes to the playbook are relevant to all Front-End developers at Springer Nature.
```
Comment (hollsk) [inline CONTRIBUTING.md]: Using the active voice makes it explicit what or who is doing the acting upon something - the passive voice only describes what is being acted upon. 

Active voice: "**A bot/slack integration** notifies #frontend when **you** merge your work..."

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -28,6 +28,6 @@ If you work at Springer Nature, please remember to keep discussion inside the is
...
 
-We hope work committed here will be of interest to the wider development community. We welcome feedback and involvement from all - so feel free to link to your latest contribution on Twitter^H^H^H^H^H^H^H^H your social media platform of choice.
+We hope the wider development community will be interested in the work committed here. We welcome feedback and involvement from all - so feel free to link to your latest contribution on Twitter^H^H^H^H^H^H^H^H your social media platform of choice.
```
Comment (hollsk) [inline CONTRIBUTING.md]: active voice: "... in the work **we** have committed here"

**Pull request merged**